src/components/inscriptionPreview/index.js

```diff
@@ -0,0 +1,66 @@
+import { useState } from "react";
+import { TailSpin } from "react-loading-icons";
+import PropTypes from "prop-types";
+import { ORDINALS_EXPLORER_URL } from "@lib/constants";
+
+export const InscriptionPreview = ({ utxo }) => {
+    const [loading, setLoading] = useState(true);
+
+    const isImage = /(^image)(\/)[a-zA-Z0-9_]*/gm.test(utxo.content_type);
```

Comment:
@habibitcoin I think u did some magic here in https://github.com/dannydeezy/nosft/pull/62 or are we good with this? Lets try to push out with turbo, since ordinals is having some issues :(

Reply:
Give https://github.com/dannydeezy/nosft/pull/62 a spin because if it works, it really calls into question if we need to switch to Turbo

I believe we were just being rate limited

Reply:
#62 wouldn't load my wallet

Reply:
Will give. full review in the morning @andredevjs but wont load non-text ordinals at the moment bc1pky7e0zer9djdxek6570226k062d2rze7e07eacx3mkyh7s9wt43qrkzwkz 

Agree, lets go with turbo now, previous PR is looking solid